contracts/HATKlerosConnector.sol

```diff
@@ -0,0 +1,711 @@
+     */
+    function changeLoserMultiplier(uint256 _loserMultiplier) external onlyGovernor {
+        loserMultiplier = _loserMultiplier;
+    }
+
+    /** @dev Changes loserAppealPeriodMultiplier variable.
+     *  @param _loserAppealPeriodMultiplier The new loserAppealPeriodMultiplier value.
+     */
+    function changeLoserAppealPeriodMultiplier(uint256 _loserAppealPeriodMultiplier) external onlyGovernor {
+        loserAppealPeriodMultiplier = _loserAppealPeriodMultiplier;
+    }
+
+    /** @dev Update the meta evidence used for disputes.
+     *  @param _metaEvidenceClaimant URI of the new meta evidence for claimant.
+     *  @param _metaEvidenceDepositor URI of the new meta evidence for depositor.
+     *  @param _metaEvidenceSubmit URI of the new meta evidence for submitting a new claim.
+     */
+    function changeMetaEvidence(string calldata _metaEvidenceClaimant, string calldata _metaEvidenceDepositor, string calldata _metaEvidenceSubmit) external onlyGovernor {
+        metaEvidenceUpdates++;
+        emit MetaEvidence(3 * metaEvidenceUpdates, _metaEvidenceClaimant);
+        emit MetaEvidence(3 * metaEvidenceUpdates + 1, _metaEvidenceDepositor);
+        emit MetaEvidence(3 * metaEvidenceUpdates + 2, _metaEvidenceSubmit);
+    }
+
+    // **************************** //
+    // *    Hacker's submission   * //
+    // **************************** //
+
+    /** @dev Allows the hacker to submit a claim through Kleros court while bypassing the committee.
+     *  @param _bountyPercentage Bounty percentage the hacker thinks he is eligible to.
+     *  @param _descriptionHash Description hash of the vulnerability.
+     *  @param _evidence Link to evidence.
+     */
+    function startProcedureToSubmitClaim(uint16 _bountyPercentage, string calldata _descriptionHash, string calldata _evidence) external payable {
+        require(_bountyPercentage <= vault.maxBounty(), "Bounty too high");
+        uint256 arbitrationCost = getArbitrationCost();
+        require(msg.value >= arbitrationCost, "Should pay the full deposit.");
+
+        uint256 localDisputeId = disputes.length;
+        uint256 metaEvidenceId = 3 * metaEvidenceUpdates + 2;
+
+        DisputeStruct storage dispute = disputes.push();
+        dispute.pendingClaimId = pendingClaims.length;
+        // Preemptively create a new funding round for future appeals.
+        dispute.rounds.push();
+
+        PendingClaim storage pendingClaim = pendingClaims.push();
+        pendingClaim.challenger = msg.sender;
+        pendingClaim.bountyPercentage = _bountyPercentage;
+        // TODO: it can be gas-consuming to store a string.
+        pendingClaim.descriptionHash = _descriptionHash;
+
+        uint256 externalDisputeId = arbitrator.createDispute{value: arbitrationCost}(RULING_OPTIONS,  arbitratorExtraData);
+        dispute.externalDisputeId = externalDisputeId;
+        externalIDtoLocalID[externalDisputeId] = localDisputeId;
```

Comment:
This mapping should be present as a part of IDisputeResolver interface. I wouldn't say it's not needed here either. Usually there is a distinction between local and external disputes in case klerosArbitrator will change

Reply:
ok